### PR TITLE
Document Avion's ability to obtain configuration automatically

### PR DESCRIPTION
**Pull request by mjg59**
**Description:**

The Avion upstream code now has support for pulling switch information from
the central server directly. Document that.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#7903

### CODE REVIEW

File: source/_components/light.avion.markdown [hunk 1]
```diff
@@ -21,6 +21,10 @@ To enable these lights, add the following lines to your `configuration.yaml` fil
 # Example configuration.yaml entry
 light:
   - platform: avion
+    username: testuser@fakedomain.com
```
Comment (Landrash) [inline source/_components/light.avion.markdown]: Basic example should only contain *Required* variables or the minimal example to get going. Feel free to also add an advanced example if so required.

File: source/_components/light.avion.markdown [hunk 2]
```diff
@@ -32,7 +36,9 @@ light:
 
 Configuration variables:
 
-- **devices**: A list of devices with their bluetooth address, a custom name to use in the frontend and the API key. The API key can be obtained by executing the following command:
```
Comment (Landrash) [inline source/_components/light.avion.markdown]: Variables are missing information about which are required and optional.

**Comment by mjg59** [comment]
@Landrash Updated, thank you!

**Comment by Landrash** [comment]
Merging since this was missed in the 0.49 merge.

**Pull request merged**